examples/device/ep/tests/utils.py

```
@@ -186,6 +186,43 @@ def __exit__(self, *_):
+
+def kineto_cuda_available() -> bool:
+    try:
+        with suppress_stdout_stderr():
+            with torch.profiler.profile(
+                activities=[torch.profiler.ProfilerActivity.CUDA]
+            ) as prof:
+                _run_cuda_profiler_sentinel()
+                torch.cuda.synchronize()
+
+        return _has_cuda_profiler_events(prof.key_averages())
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)